agr_java_core/src/main/java/org/alliancegenome/core/variant/converters/AlleleVariantSequenceConverter.java

```diff
@@ -31,27 +31,31 @@
         }
 
         SOTerm variantType = new SOTerm();
-        variantType.setName(ctx.getType().name().toUpperCase());
-        variantType.setPrimaryKey(ctx.getType().name());
-        if ("INDEL".equals(ctx.getType().name())) {
-            variantType.setName("delins");
-            variantType.setPrimaryKey("delins");
+
+        if(!"SYMBOLIC".equalsIgnoreCase(ctx.getType().name()) && !"MIXED".equalsIgnoreCase(ctx.getType().name())){
```

Comment:
Does this have to do a `equalsIgnoreCase` or can we use a straight `equals` ?

Reply:
I'm sure equals would work. I probably did that as an extra safeguard just in case.